buildSrc/src/main/groovy/io.deephaven.javadoc-conventions.gradle

```diff
@@ -0,0 +1,18 @@
+// Neither plain withType(Javadoc) {..} nor withType(Javadoc).all {..} will work here, somehow
+// this breaks sourcesets in some projects.
+tasks.withType(Javadoc).configureEach {
+    javadocTool = javaToolchains.javadocToolFor{javadocSpec ->
+        // Javadoc version >=11 is needed for search
+        // Javadoc version >12 is needed to avoid javadoc bugs in linking with modules
+        javadocSpec.languageVersion = JavaLanguageVersion.of(17)
+    }
+
+    options.encoding = 'UTF-8'
+
+    options.links = ['https://docs.oracle.com/en/java/javase/11/docs/api/']
+    options.addBooleanOption('Xdoclint:none', true)
```

Comment:
Dropping `-quiet` was intentional?

Reply:
Yes, it did nothing, it was a workaround for a gradle javadoc limitation. Boolean is probably more accurate or at least less confusing, so no one thinks that `-quiet` was intended to be added as a real flag.